edi/views/edi_exchange_record_views.xml

```diff
@@ -23,28 +23,65 @@
             <form string="EDI Exchange Record">
                 <field name="direction" invisible="1" />
                 <field name="retryable" invisible="1" />
-                <header>
-                    <button
-                        name="action_open_related_record"
-                        type="object"
-                        string="Related record"
-                        attrs="{'invisible': [('res_id', '=', False)]}"
-                    />
-                    <!-- FIXME: this `invisible` domain does not work -->
-                    <button
-                        name="action_open_related_exchanges"
-                        type="object"
-                        string="Related exchanges"
-                        attrs="{'invisible': [('related_exchange_ids', '=', False)]}"
-                    />
-                    <button
-                        name="action_retry"
-                        type="object"
-                        string="Retry"
-                        attrs="{'invisible': [('retryable', '=', False)]}"
-                    />
-                </header>
+                <field name="exchange_file_auto_generate" invisible="1" />
+                <header />
                 <sheet>
+                    <div name="button_box" class="oe_button_box">
+<button
+                            name="action_exchange_send"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-send"
+                            attrs="{'invisible': ['|', ('edi_exchange_state', '!=', 'output_pending'), ('exchange_file_auto_generate', '=', True)]}"
```

Comment:
the send should be independent from `exchange_file_auto_generate`. This flag is only for generating the output.